definition/IVoipServerConfig.ts

```diff
@@ -2,11 +2,24 @@ import { IRocketChatRecord } from './IRocketChatRecord';
 
 export enum ServerType {
 	MANAGEMENT = 'management',
-	CALLS = 'calls',
+	CALL_SERVER = 'call-server',
 }
 
 export interface IVoipServerConfig extends IRocketChatRecord {
 	type: ServerType;
 	host: string;
+	configData: IManagementConfigData | ICallServerConfigData;
+}
+
+export interface IManagementConfigData {
 	port: number;
+	serverName: string; // Internal name to be used in reports
```

Comment:
serverName will be in IVoipServerConfig. This is a user defined name. This will be used in the reports to identify the server.

Reply:
Okay got it. Let me move this property to the parent interface i.e. IVoipServerConfig